wagtailio/project_styleguide/templates/patterns/components/hero/hero.html

```diff
@@ -1,18 +1,23 @@
+{% load wagtailcore_tags %}
+
 <div class="grid">
     <div class="hero">
         <div class="hero__inner">
             <h1 class="hero__heading">{{ value.heading }}</h1>
 
-            {% if value.subheading %}
-                <h2 class="hero__subheading intro-big">{{ value.subheading }}</h2>
+            {% if value.sub_heading %}
+                <h2 class="hero__subheading intro-big">{{ value.sub_heading }}</h2>
             {% endif %}
 
             {% if value.intro %}
-                <p class="hero__intro">{{ value.intro }}</p>
+                <p class="hero__intro">{{ value.intro|richtext }}</p>
             {% endif %}
 
-            {% if value.cta_url or value.cta_page %}
-                {% include "patterns/components/buttons/button.html" with arrow=True url=url title=value.cta_text classes="hero__button" %}
+            {% if value.cta %}
```

Comment:
Note: Unless I'm mistaken, you can just you the for loop below for this, if the cta value is `None` the loop will not output anything.

Reply:
Thanks, done in ae8d891